web/src/beta/lib/reearth-ui/components/Modal/index.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { Button } from "../Button";
+import { ModalPanel } from "../ModalPanel";
+
+import { ModalProps, Modal } from ".";
+
+const meta: Meta<ModalProps> = {
+  component: Modal,
+};
+
+export default meta;
+type Story = StoryObj<typeof Modal>;
+
```

Comment:
For stories of Modal i think we need to mock up a trigger to show the full UX of modal. (trigger open and close)